.github/workflows/workflow.yaml

```diff
@@ -8,6 +8,14 @@ on:
   workflow_dispatch: # allows manual triggering
 
 jobs:
+  cancel-running-workflows:
+    name: Cancel running workflows
+    runs-on: ubuntu-20.04
+    steps:
+      - name: cancel running workflows
+        uses: styfle/cancel-workflow-action@0.11.0
+        with:
+          access_token: ${{ github.token }}
```

Comment:
Nice! Usually I tried to cancel these manually, when I was aware of them. 

But could you use `ubuntu-latest` please, so the jobs are run on the same image?

And I would prefer this to be split out into a separate commit.

Reply:
Switched to `ubuntu-latest` and added a separate commit.